unittests/deep_mind_tests.cpp

```diff
@@ -49,7 +49,7 @@ struct deep_mind_log_fixture
 
 struct deep_mind_tester : deep_mind_log_fixture, validating_tester
 {
-   deep_mind_tester() : validating_tester({}, &deep_mind_logger) {}
+   deep_mind_tester() : validating_tester({}, &deep_mind_logger, setup_policy::full_but_disable_deferrd_trx) {}
```

Comment:
Why do the deep_mind_tests require deferred transactions to not be disabled?

Reply:
`producer_block_id` differs between different sets of activated protocol features, due to actions to activate the protocol features are different.